qmxgraph/page/api.js

```diff
@@ -1175,6 +1175,10 @@ graphs.Api.prototype.onViewUpdate = function onViewUpdate (handler) {
     var listener = (function(sender, evt) {
         handler(this.dump(), this.getScaleAndTranslation());
     }).bind(this);
+    graph.getView().addListener(mxEvent.SCALE, listener);
```

Comment:
Did you check if those overlap  with any undo event?

Reply:
Didn't see any weird behavior going on. Not sure how else I could make extra checks.